src/components/Components/DropFile.tsx

```diff
@@ -112,20 +116,25 @@ const FileDropArea = observer(() => {
                 const api_url = 'https://eudfxg3a9l.execute-api.us-west-2.amazonaws.com/dev/'
                 axios.post(api_url, data).then(response => {
                   const gltf_url = response.data['url']; .replace(/\.\w+$/, '.gltf')
-                  appState.setCurrentModelPath(gltf_url); */
+                  appState.setCurrentModelPath(gltf_url); 
+                  CognitoIdentityServiceProvider.6jlm2jeibh9aqb0dg34q2uf8pu.ayman1234.userData*/
```

Comment:
This looks like some kind of ID or sensible information. Should we remove it?

Reply:
Please remove, good catch, same for the other comment.